src/Neo/Plugins/Plugin.cs

```diff
@@ -189,7 +189,24 @@ private static void LoadPlugin(Assembly assembly)
             }
         }
 
-        internal static void LoadPlugins()
+        public static void LoadPlugins(string[] pluginNames)
+        {
+            if (!Directory.Exists(PluginsDirectory)) return;
+            foreach (var rootPath in Directory.GetDirectories(PluginsDirectory))
```

Comment:
Better to check in the above method if it's contained, in this array.

Reply:
This way we don't have to load the `assemblies/dlls` (more secure) and we don't change the current logic.

Reply:
> we don't change the current logic.

It's changed, you moved when the plugins are loaded

Reply:
Well, yes. You have to call `Plugin.LoadPlugins` directly now or else no plugins will be loaded. Plus I had problems in `new-cli` with plugins not being loaded because `Plugins.LoadPlugins` is called on  `static` `NeoSystem's` constructor. So if you dont call it `statically` they don't load and the call is `internal`.

Reply:
I prefer not to alter the usual behavior and just filter them in the current method